docs/cpp/config-wsl.md

```diff
@@ -57,16 +57,20 @@ To successfully complete this tutorial, you must do the following steps:
 
 ## Create a workspace
 
-At a Windows command prompt, create an empty folder called `projects` (just like you did on Ubuntu) where you can place your helloworld project, and other VS Code projects you might create. Then create a subfolder called `helloworld`, navigate into it, and open VS Code (`code`) in that folder (`.`) by entering the following commands:
+In Windows, you will need an empty folder called `projects` (just like you now have on Ubuntu) where you can place your helloworld project, and other VS Code projects you might create. 
 
-    ```bash
```

Comment:
The code block language bash should not have been displayed since it is just for syntax highlighting but it looks like 4 leading spaces versus 3 confused the Markdown parser. We usually use bash for all command line code blocks and I didn't know cmd was an allowed language identifier. cmd seems to render well (commands in red).

![image](https://user-images.githubusercontent.com/12818376/56067227-44c33280-5d2f-11e9-903b-83f38dea16d1.png)